src/GEMItem.h

```diff
@@ -164,15 +177,18 @@ class GEMItem {
       @param 'getValueStr' - pointer to function that will return a char* and is executed when the item is updated
     */
     GEMItem(const char* const title_, const char* const (*getValueStr)(void));
+    GEMItem(const char* const (*getValueStr)(void));
     /* 
       Constructor for menu item that represents a single static string as text entry,
       @param 'title_' - title of the menu item displayed on the screen
     */
     GEMItem(const char* const title_);
-    /*
-      Provide any GEMItem constructor with __FlashStringHelper (for F("asd") macro) instead of char* title.
-    */
-    template<typename... Args> GEMItem(const __FlashStringHelper* title_, Args... args);
+
+    
+    // /*
+    //   Provide any GEMItem constructor with __FlashStringHelper (for F("asd") macro) instead of char* title.
+    // */
+    // template<typename... Args> GEMItem(const __FlashStringHelper* title_, Args... args);
```

Comment:
Have you removed this whole template and uses it no longer?

Reply:
I have removed it